### ISSUE TITLE
Craft 5 Compatibility

### ISSUE THREAD

**Issue by AndrewMac** [mediabeastnz/craft-commerce-back-in-stock#29]
Hello there!
Just wondering if you have any timeline on updating this for Craft Commerce 5?
And perhaps also your Abandoned Cart plugin? 
Thanks heaps, Andrew

**Comment by Romanavr** [comment]
Same request. Any ETA?

**Comment by jamiegrisdale** [comment]
@mediabeastnz I'd like to know the same please

**Comment by mylesthedev** [comment]
Hey all,

Sorry my time in Craft world has been limited for a while. I'll take a look to see what's involved in getting a version out for C5 support asap!

**Comment by boboldehampsink** [comment]
+1

**Comment by wmdhosting** [comment]
any news on this

**Comment by engram-design** [comment]
Hey @mediabeastnz! Just to chime in, we've added this to our [Craft 4-5 migration project](https://verbb.io/craft-4), as people are keen to see this plugin on Craft 5.

If you're no longer developing this plugin, we'd be happy to continue development at Verbb (including updating to Craft 5) - but just let us know if you've got plans to continue things and update to Craft 5!

**Comment by mylesthedev** [comment]
@engram-design that would be awesome if you have time. I'm in a different ecosystem as of late so limited Craft time.

Let me know if you need me to do anything on my end

**Comment by engram-design** [comment]
Fair enough Myles! We'd require the plugin and repo be transferred to Verbb, and we'd take it from there. But again, no pressure if you'd still like to retain ownership of this plugin if you get back to the Craft scene in the future — your call!

If you're happy for us to take it off your hands, you can transfer it to my personal `engram-design` account, as it's otherwise a bit tricky to transfer straight to the `verbb` GitHub Organisation.

**Comment by engram-design** [comment]
Thanks to @mediabeastnz for transferring this to Verbb, we'll get started on migrating this one to Craft 5 in the coming weeks!

**Comment by AndrewMac** [comment]
Thanks to both of you, @mediabeastnz and @engram-design. It's great news!  Can I ask, though, if you've come to a similar kind of arrangement with the Abandoned Cart plugin?

**Comment by jamiegrisdale** [comment]
I was wondering the same

**Comment by engram-design** [comment]
All too happy to take over that plugin as well!

**Comment by boboldehampsink** [comment]
@engram-design any ETA on this?

**Comment by engram-design** [comment]
Looking to do a bunch of these Craft 5 upgrades at once, should expect them in the next month or two.

**Comment by engram-design** [comment]
Updated in [4.0.0](https://github.com/verbb/back-in-stock/releases/tag/4.0.0) - old news now!
